packages/react-slider/src/components/Slider/stories/SliderSmall.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
```

Comment:
Should we have this story be `SliderSizes` instead of `SliderSmall` and show all the sizes of `Slider` next to each other?

Reply:
Good call. Usually each name is the prop itself. Would make even more sense if we had a 3rd size, but better to keep consistent

Reply:
fixed